database/data.js

```diff
@@ -0,0 +1,87 @@
+var faker = require('faker');
+var csv = require('csv-parser');
+var fs = require('fs');
+
+//set seed
+faker.seed(123);
+
+var generateRating = function(max){
+  // return random Number between 0 and max
+  var randomNum = parseInt(Math.random() * max)
+  return randomNum;
+};
+
+
+// generate data
+var data_bookInfo = function(){
```

Comment:
I recommend more parallelism w/ your function names.

If you generate ratings with a function called `generateRating()`, perhaps you should generate book info with a function called `generateBookInfo()`.

Reply:
parsed through code and made function names more expressive ->  `generateBookInfo` , `generateUsers`,`generateReadStatus`